content/language/R/develop.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Use containers for R development
+keywords: R, local, development
+description: Learn how to develop your R application locally.
+---
+
+## Prerequisites
+
+Complete [Containerize a R application](containerize.md).
+
+## Overview
+
+In this section, you'll learn how to set up a development environment for your containerized application. This includes:
+
+- Adding a local database and persisting data
+- Configuring Compose to automatically update your running Compose services as you edit and save your code
+
+## Get the sample application
+
+You'll need to clone a new repository to get a sample application that includes logic to connect to the database.
+
+1. Change to a directory where you want to clone the repository and run the following command.
+
+   ```console
+   $ git clone https://github.com/mfranzon/r-docker-dev.git
+   ```
+
+2. In the cloned repository's directory, run `docker init` to create the necessary Docker files. Refer to the following example to answer the prompts from `docker init`.
```

Comment:
Again a Dockerfile and a compose.yaml already exists in the repository. I don't think we can use `docker init` for this project.

Reply:
Removed `docker init`